INVENTORY.md

```diff
@@ -0,0 +1,22 @@
+# Inventory
+
+this page talks about common inventory information information provided by DPU vendors in a same format
+
+- Questions:
+  - Where does it run?
+    - service/container running on ARM cores?
+    - DPU firmware?
+    - DPU Bootloaders?
+    - DPU BMC?
+  - What information does it provide ?
+    - Vendor, SN/PN, ...
+    - Credentials
+    - Virtual location in the host (mother board slot or PCIe BDF?)
+    - Capabilities (cou, mem, offloads,...)
+    - What protocols and/or privisoning methods are supported
+    - What DPU belongs to what Host ?
```

Comment:
Is this implying we're supporting a model where DPUs are external to a host and connected into the host remotely, opening up the case where a single DPU could be connected to multiple hosts? Or are we blurring what a single agent on a DPU is collecting/reporting compared to what a distributed controller would be reporting?

Reply:
sometimes there are 1 or more DPUs in the host... some companies what to know in this case what DOU belongs to what host and what is the location or index of the DPU in that host... we do want a single DPU to know about the host... 

sometimes DPUs are standalone without any host behind it and we would like also to know that...